qa-tests-backend/src/test/groovy/UpgradesTest.groovy

```diff
@@ -253,6 +253,12 @@ class UpgradesTest extends BaseSpecification {
             "f09f8da1-6111-4ca0-8f49-294a76c65115" : new KnownPolicyDiffs().setPolicyAsDisabled(),
             "a919ccaf-6b43-4160-ac5d-a405e1440a41" : new KnownPolicyDiffs().setPolicyAsEnabled(),
             "93f4b2dd-ef5a-419e-8371-38aed480fb36" : new KnownPolicyDiffs().setPolicyAsDisabled(),
+            "ed8c7957-14de-40bc-aeab-d27ceeecfa7b":  new KnownPolicyDiffs()
```

Comment:
You can remove this change since it's not a known diff but rather a bug (that you fixed)